gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,235 @@
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies the header subtitle content placed under the header title.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>headerParagraph</NxTable.Cell>
+                <NxTable.Cell>ReactNode | string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies the header paragraph content placed under the subtitle if the
+                  subtitle is specified, if not, it will be placed under the header title.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>HTML <NxCode>&lt;dialog&gt;</NxCode> Attributes</NxTable.Cell>
+                <NxTable.Cell>
+                  <NxTextLink external href="https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes">
+                    HTML Attributes
+                  </NxTextLink>
+                </NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  NxDrawer supports any html attribute that's normally supported by
+                  {' '}<NxCode>&lt;dialog&gt;</NxCode> elements.
+                </NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Body>
+          </NxTable>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>Using NxDrawer with nx-global-header</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            When using <NxCode>NxDrawer</NxCode> with <NxCode>nx-global-header</NxCode>,{' '}
+            <NxCode>nx-drawer--with-global-header</NxCode> className must be applied so the
+            drawer does not overlap with the global header.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>Built-in Header</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            The header is placed at the top of the drawer component. It consists of four parts:
+            the close button, the title, the subtitle (optional), and the paragraph (optional).
+            The title, subtitle, and paragraph can be specified via the <NxCode>headerTitle</NxCode>,{' '}
+            <NxCode>headerSubtitle</NxCode>, and <NxCode>headerParagraph</NxCode> props respectively.
+          </NxP>
+        </NxTile.Subsection>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3>NxDrawer.Content (Required)</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxP>
+            This is a convenient component that specifies the main body content of the drawer. It is placed
```

Comment:
Instead of "This is a convenient component that specifies the main body content of the drawer.", how about "This component specifies the drawer's body content."?

Reply:
> Instead of "This is a convenient component that specifies the main body content of the drawer.", how about "This component specifies the drawer's body content."?

Fixed here: 139ae1d5bb5f3141dd1154501a7dff94e2a046d3